config/initializers/constants.rb

```diff
@@ -11,6 +11,6 @@
 TUMBLR_BASE          = 'doubleunion.tumblr.com'
 TUMBLR_URL           = "http://#{TUMBLR_BASE}"
 
-MAILING_LIST_GENERAL = "http://lists.doubleunion.org/listinfo.cgi/doubleunion-doubleunion.org"
+MAILING_LIST_GENERAL = "https://groups.google.com/a/doubleunion.org/forum/#!forum/public"
```

Comment:
Yay! I think this is great to pull out into a contstant since it's a pretty important link.